site/en/docs/privacy-sandbox/unified-origin-trial/index.md

```diff
@@ -0,0 +1,372 @@
+    </tr>
+    <tr>
+      <td>
+<a
+href="/docs/privacy-sandbox/topics-experiment/">Topics</a></td>
+      <td>
+<a
+href="/docs/privacy-sandbox/topics-experiment/">Developer
+guide for the origin trial</a>. Topics was briefly disabled in the
+origin trial due to a <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=1321140">bug that affected browser stability.</a></td>
+    </tr>
+    <tr>
+      <td>
+<a
+href="/docs/privacy-sandbox/fenced-frame/">Fenced
+Frames</a></td>
+      <td>
+Fenced Frames added to the origin trial. See the FLEDGE guidance
+for experiment usage.</td>
+    </tr>
+  </tbody>
+</table>
+
+#### April 2022
+
+Origin trial begins with a [limited proportion of users from Chrome 102 Beta](https://groups.google.com/a/chromium.org/g/blink-dev/c/Vi-Rj37aZLs/m/wzeBWfxxEgAJ).  
+APIs included in the trial:
+
+<table class="with-heading-tint fixed-table width-full">
+  <thead>
+    <tr>
+      <th>
+<strong>API</strong></th>
+      <th>
+<strong>Notes</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>
+<a
+href="/docs/privacy-sandbox/attribution-reporting-experiment/">Attribution
+Reporting</a></td>
+      <td>
+Individual <a
+href="https://groups.google.com/a/chromium.org/g/blink-dev/c/jEnNpideO1Y/m/nlEDdjmnCgAJ">Intent
+to Experiment (I2E)</a> post. Attribution Reporting also available as
+an individual origin trial.</td>
+    </tr>
+    <tr>
+      <td>
+<a
+href="/docs/privacy-sandbox/fledge-experiment/">FLEDGE</a></td>
+      <td>
+Individual <a
+href="https://groups.google.com/a/chromium.org/g/blink-dev/c/0VmMSsDWsFg/m/_0T5qleqCgAJ">Intent
+to Experiment (I2E)</a> post. FLEDGE available on desktop only with a
+<a
+href="https://github.com/WICG/turtledove/blob/main/Proposed_First_FLEDGE_OT_Details.md">subset
+of functionality</a>.</td>
+    </tr>
+    <tr>
+      <td>
+<a
+href="/docs/privacy-sandbox/topics-experiment/">Topics</a></td>
+      <td>
+Individual <a
+href="https://groups.google.com/a/chromium.org/g/blink-dev/c/oTwd6VwCwqs/m/jPkW3T-mCgAJ">Intent
+to Experiment (I2E)</a> post.</td>
+    </tr>
+  </tbody>
+</table>
+{% endDetails %}
+
+## Sign up for and configure the origin trial {: #configure}
+
+To activate the origin trial on your site, you will need to register
+and embed the assigned origin trial token (a time-based string
+for your specific access to the origin trial). Learn more in the
+[Getting started with Chrome's origin trials](/blog/origin-trials/).  
+
+{% Aside %}
+Register for the [Privacy Sandbox Relevance and Measurement trial](/origintrials/#/view_trial/771241436187197441). 
+{% endAside %}
+
+Origin trial tokens are granted immediately, and you can revoke or recreate them
+at any time. 
+
+For every page where you want to use the origin trial, you will need to include a trial token with that specific page's HTML or response.
+
+Use a `<meta>` tag in the top-level page's `<head>`: 
+	
+`<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`. 
```

Comment:
A GitHub suggestion won't work, *but* this should be styled as a fenced block instead of in-line code:

https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

The language is html, so after the opening three backticks, include html with no space.